### PR TITLE
Set class to menu item when active

### LINKED ISSUE #81: apply class="active" to currently active page/section
Not entirely sure how to achieve this. But the current behaviour is the only indication of page is the breadcrumb.

The Bootstrapious theme has the behaviour of active pages, but that is hard coded, not sure how to do the same with Hugo.

Maybe some sort of conditional logic?

### PR DESCRIPTION
**Pull request by adrianmo**
- Fixes #81
- Menu items for "contact" and "faq" have been moved to the respective contect (Markdown) files. It wasn't working otherwise, not sure why.
- Top level pages work fine, but individual blog posts do not highlight the "Blog" menu item. Need to investigate.

### CODE REVIEW

**Comment by adrianmo** [comment]
We still need to make it work with individual posts like `/blog/2015/10/10/migrate-from-jekyll/`.

**Comment by GeorgeWL** [comment]
maybe (pseudo code)

``` if {{section}} = blog {
class = active
}
```

**Comment by adrianmo** [comment]
I agree - I'll dig into it a bit more and have a look at other Hugo sites and see how they implement it.

File: exampleSite/config.toml [hunk 1]
```diff
@@ -26,17 +26,6 @@ paginate = 10
     url  = "/blog/"
     weight = 2
 
-[[menu.main]]
```
Comment (ryanfox1985) [inline exampleSite/config.toml]: @adrianmo why do you move this part? And you remove some keys.

**Comment by salim-b** [comment]
Since #191 is merged now, I strongly suggest to close this pull request, it seems completely obsolete now.

**Comment by ryanfox1985** [comment]
I agree

**Pull request closed without merging**